include/stronk/stronk.h

```diff
@@ -61,6 +61,28 @@ struct stronk : public Skills<Tag>...
         return this->_you_should_not_be_using_this_but_rather_unwrap;
     }
 
+    template<typename ExpectedT, typename ExpectedUnderlyingT>
+    [[nodiscard]]
+    constexpr auto unwrap_as() noexcept -> ExpectedUnderlyingT&
```

Comment:
Consider adding a comment about what the unwrap_as functions are used for.

I suppose from the tests that it is to unwrap multiple layers of stronk types?

Reply:
And potentially as a safety mechanism, for when its important that the underlying type does not change.